src/devtools/shared/contextmenu.js

```diff
@@ -10,55 +10,6 @@ function inToolbox() {
   }
 }
```

Comment:
surprised we can't kill this entire file

Reply:
It's still being used in a couple places when I checked